File: api/account.js
export default async function handler(req, res) {
  res.setHeader('Access-Control-Allow-Origin', '*');
  try {
    const base = process.env.ALPACA_PAPER === 'false'
      ? 'https://api.alpaca.markets'
      : 'https://paper-api.alpaca.markets';
    const r = await fetch(`${base}/v2/account`, {
      headers: {
        'APCA-API-KEY-ID': process.env.ALPACA_API_KEY || '',
        'APCA-API-SECRET-KEY': process.env.ALPACA_SECRET_KEY || '',
      },
    });
    const d = await r.json();
    if (!r.ok) return res.json({ ok: false, error: d.message || 'Alpaca error' });
    res.json({
      ok: true,
      portfolio_value: d.portfolio_value,
      equity: d.equity,
      cash: d.cash,
      buying_power: d.buying_power,
      daytrade_count: d.daytrade_count,
    });
  } catch (e) {
    res.json({ ok: false, error: e.message });
  }
}

File: api/account.py
import os, json
from http.server import BaseHTTPRequestHandler


def _get_data():
    from alpaca.trading.client import TradingClient
    client = TradingClient(
        api_key=os.environ["ALPACA_API_KEY"],
        secret_key=os.environ["ALPACA_SECRET_KEY"],
        paper=os.environ.get("ALPACA_PAPER", "true").lower() == "true",
    )
    a = client.get_account()
    return {
        "ok": True,
        "equity": str(a.equity),
        "cash": str(a.cash),
        "buying_power": str(a.buying_power),
        "portfolio_value": str(a.portfolio_value),
        "daytrade_count": a.daytrade_count,
    }


class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        try:
            body = json.dumps(_get_data()).encode()
        except Exception as e:
            body = json.dumps({"ok": False, "error": str(e)}).encode()
        self.send_response(200)
        self.send_header("Content-Type", "application/json")
        self.send_header("Access-Control-Allow-Origin", "*")
        self.end_headers()
        self.wfile.write(body)

    def log_message(self, *args):
        pass
